### DISCUSSION TITLE
Prototype causes VoiceOver to read out one word at a time

### DISCUSSION THREAD

**Discussion by abmoo2** [latex3/tagging-project#1509] [Issues with accessibility checkers and other AT software]
Hi team,

I confess that my knowledge of LaTeX is infinitesimal. I work in accessibility, though, and I am _so_ excited to see this development. My advice for folks on what to do with their LaTeX documents has been sadly quite limited. (If you switch to sans serif fonts, that helps; if you somehow use no packages at all, you can use Pandoc to convert it to HTML, but everyone uses packages; etc.)

I was just listening to the document that Mittelbach and Fischer created with the prototype. I noticed that in VoiceOver, at least, it was reading one word at a time, so the document sounded like this: "This. paper. was. initially. presented. at. the. 5th. Interna.
tional. Workshop. on. open quotation mark. Digitization...." It even broke up 'international' because it was split over two lines.

I wonder if part of the development plan is to help the software read it more naturally? I worry that a long document being read out in that manner would be prohibitively difficult to comprehend.

Again, I'm so excited to see that someone is doing this work, and I can't wait to see the end product. My feedback is in the spirit of making the thing better! Thank you!

**Comment by mbertucci47** [comment]
I converted this to a discussion since it is about specific AT software
